include/arch/arm-imx/limits.h

```diff
@@ -16,11 +16,45 @@
 #ifndef _LIBPHOENIX_ARCH_ARM_IMX_LIMITS_H_
 #define _LIBPHOENIX_ARCH_ARM_IMX_LIMITS_H_
 
+#include "arch.h"
+
+#define SCHAR_MIN -128
+#define SCHAR_MAX 127
+#define UCHAR_MAX 255
 
 #define CHAR_MIN 0
+#define CHAR_MAX UCHAR_MAX
+#define CHAR_BIT 8
+
+#define MB_LEN_MAX 4
+
+#define SHRT_MIN -32768
+#define SHRT_MAX 32767
+#define USHRT_MAX 65535
+
+#define INT_MIN -2147483648
+#define INT_MAX 0x7fffffff
+#define UINT_MAX 0xffffffff
+
+#define LONG_MIN INT_MIN
+#define LONG_MAX INT_MAX
+#define ULONG_MAX UINT_MAX
+
+#define LONG_LONG_MIN 0x8000000000000000ll
```

Comment:
Use the LL specifier to be consistent with rest of the code.

Reply:
Rationale: l looks very similar to 1 on some fonts.

Reply:
Agree, fixed.